_spikes-poc/mssql-test/routes/index.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const router = express.Router();
+const sqlService = require('../sql.service')
+const R = require('ramda')
+const data = 'sample-data '.repeat(1000)
+
+/* GET home page. */
+router.get('/', async function(req, res, next) {
+  try {
+    const result3 = await sqlService.query('SELECT COUNT(*) FROM [mtc_admin].[sessions]')
```

Comment:
whats the point of storing the first and last db transactions to a const if you are not using them?
I suppose you just replace these variables on your code below?
What is the goal here?

Reply:
This is just a spike.  Code was being moved around, whilst I made many attempts to get a closed database handle open again.  It never happened.